### DISCUSSION TITLE
I cannot set dropdown position to below again on mantine 6 and above just the way we can do on mantine 5

### DISCUSSION THREAD

**Discussion by Danztee** [mantinedev/mantine#5661] [Q&A]
### Dependencies check up

- [X] I have verified that I use latest version of all @mantine/* packages

### What version of @mantine/* packages do you have in package.json?

6.0.13

### What package has an issue?

@mantine/dates

### What framework do you use?

Next.js

### In which browsers you can reproduce the issue?

Chrome

### Describe the bug

I want to be able to add the dropdown position of the date calendar to the bottom... it's going to the top and not even scrolling

### Self-service

- [X] I would be willing to implement a fix for this issue

**Comment by rtivital** [comment]
You can pass props down to the Popover component with `popoverProps`
